Client/Ecosia/UI/APNConsent/APNConsentViewModelProtocol.swift

```diff
@@ -0,0 +1,31 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+// MARK: - APNConsentViewModelProtocol
+
+/// Protocol defining the properties required for an APN consent view model.
+protocol APNConsentViewModelProtocol {
+    var listItems: [APNConsentListItem] { get }
+    var title: String { get }
+    var image: UIImage? { get }
+    var ctaAllowButtonTitle: String { get }
+    var skipButtonTitle: String { get }
+}
+
+extension APNConsentViewModelProtocol {
+    
+    // MARK: Default values for optional properties
+    
+    /// Default implementation for the CTA (Call to Action) allow button title.
+    var ctaAllowButtonTitle: String {
```

Comment:
Were this added as default because they don't depend on the variant? If that is the case I feel this could be moved to view as there is no business logic or at least live with the others to have it all together, what do you think?